code/_onclick/drag_drop.dm

```diff
@@ -5,14 +5,14 @@
 	recieving object instead, so that's the default action.  This allows you to drag
 	almost anything into a trash can.
 */
-/atom/MouseDrop(atom/over)
+/atom/MouseDrop(var/atom/over, src_location, over_location, src_control, over_control, params)
```

Comment:
Properly propagating the various native params through mousedrops too
This allows things to be dragged and dropped onto tables at precise pixel coords